content/documentation/pages/4-batch-developer-guides/1-getting-started.md

```diff
@@ -11,26 +11,61 @@ This content can come from the current 'getting started' guide.
 
 Explain the OOTB apps.
```

Comment:
Give a brief description of OOTB apps and discuss what timestamp does.    Then remove the rest of the comments above.